access-common/src/main/java/edu/unc/lib/dl/ui/access/StoreAccessLevelFilter.java

```diff
@@ -73,17 +73,17 @@ public void doFilterInternal(HttpServletRequest request, HttpServletResponse res
                 session.removeAttribute("accessLevel");
                 accessLevel = new AccessLevel(username);
                 session.setAttribute("accessLevel", accessLevel);
-                AccessGroupSet groups = GroupsThreadStore.getGroups();
-                if (globalPermissionEvaluator.hasGlobalPrincipal(groups)) {
-                    Set<UserRole> roles = globalPermissionEvaluator.getGlobalUserRoles(groups);
+                AccessGroupSet principals = GroupsThreadStore.getAgentPrincipals().getPrincipals();
```

Comment:
Not a big issue, but would a wrapper method be useful for all of these `GroupsThreadStore.getAgentPrincipals().getPrincipals()` calls?